subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -80,15 +84,33 @@ static int publish_get_payload(struct mqtt_client *client,
 	return 0;
 }
 
+static int construct_job_id_get_topic(const u8_t *client_id, const u8_t *job_id,
+				      const u8_t *suffix, u8_t *topic_buf)
+{
+	__ASSERT_NO_MSG(client_id != NULL);
+	__ASSERT_NO_MSG(topic_buf != NULL);
+	int ret = snprintf(topic_buf, JOB_ID_GET_TOPIC_MAX_LEN,
+			   JOB_ID_GET_TOPIC_TEMPLATE, client_id, job_id
```

Comment:
You miss a comma after `job_id` here. I think that's what bothers the CI.

Reply:
wow, thanks. I really missed that, thank you so much I relied too much on my syntax highlighting to catch those errors seems they don't handle break lines so well. Thank you so much for spotting it.